packages/core/js-client/src/network.ts

```diff
@@ -0,0 +1,190 @@
+/**
+ * Copyright 2023 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export type Node = { peerId: string; multiaddr: string };
```

Comment:
@folex really doesn't like the name Node in this context. I pretty much agree. If we move it here maybe we should name it differently. Also we discussed that maybe exposoing only multiaddrs would be enough and a simple function to get peerid out of them (seems like js people would rarelly need peerId directly, I don't know)

Also since now CLI generates a file that contains all these adresses - maybe there is no reason to expose this from the client at all anymore, since you would have to change your source code each time you switch environments, but with CLI controlling this part - you wouldn't have to do that and would only have to import one file